client/src/app/simplelogin/simplelogin.component.ts

```diff
@@ -15,37 +15,43 @@
  */
 import { environment } from '../../environments/environment';
 import { Component, OnInit } from '@angular/core';
+import { BackendService } from '../backend.service';
+import { SimpleLoginRequest } from '../simple-login-request';
 
 @Component({
   selector: 'app-simplelogin',
   templateUrl: './simplelogin.component.html',
   styleUrls: ['./simplelogin.component.css']
 })
+
 export class SimpleLoginComponent implements OnInit {
   url = environment.apiPostUrl;
   username: string = "";
   password: string = "";
   enableSubmit: boolean = false;
   error_text: string = "";
-  constructor() { }
+  constructor(private service : BackendService) { }
```

Comment:
Run ng test and e2e tests to make sure we don't have to update anything to make them pass. Travis should pick up the failure, if there is one, too.

Reply:
Done.